cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -341,6 +348,29 @@ void CobTwistController::solveTwist(KDL::Twist twist)
     int ret_ik;
     KDL::JntArray q_dot_ik(chain_.getNrOfJoints());
 
+    if(firstIteration_)
+    {
+        time_now_ = ros::Time::now();
+        last_update_time_ = time_now_;
+        integration_period_ = time_now_ - last_update_time_;
+    }
+    else
+    {
+        time_now_ = ros::Time::now();
+        integration_period_ = time_now_ - last_update_time_;
+    }
+
+    try
+   {
+       tf_listener_.waitForTransform("base_link",chain_tip_link_, ros::Time(0), ros::Duration(0.5));
+       tf_listener_.lookupTransform("base_link",chain_tip_link_,  ros::Time(0), bl_transform_ct);
+   }
+   catch (tf::TransformException &ex)
+   {
+       ROS_ERROR("%s",ex.what());
+       return;
+   }
```

Comment:
where does this listener come from?
I guess it's a merge left-over...please remove...